src/SmartComponents/PolicyDetails/PolicyDetails.js

```diff
@@ -30,13 +30,50 @@ import PolicyMultiversionRules from './PolicyMultiversionRules';
 import './PolicyDetails.scss';
 import useSaveValueToPolicy from './hooks/useSaveValueToPolicy';
 import usePolicyQuery from 'Utilities/hooks/usePolicyQuery';
+import usePolicyQuery2 from '../../Utilities/hooks/usePolicyQuery/usePolicyQuery2';
+import useAPIV2FeatureFlag from '../../Utilities/hooks/useAPIV2FeatureFlag';
 
-export const PolicyDetails = ({ route }) => {
-  const defaultTab = 'details';
+export const PolicyDetailsWrapper = ({ route }) => {
+  const apiV2Enabled = useAPIV2FeatureFlag();
+  console.log(apiV2Enabled);
+
+  return apiV2Enabled ? (
+    <PolicyDetailsV2 route={route} />
+  ) : (
+    <PolicyDetailsGraphQL route={route} />
+  );
+};
+
+const PolicyDetailsGraphQL = ({ route }) => {
   const { policy_id: policyId } = useParams();
-  const { data, error, loading, refetch } = usePolicyQuery({
+  const query = usePolicyQuery({
     policyId,
   });
+
+  console.log({ data: query?.data });
+
+  return <PolicyDetailsBase query={query} route={route} />;
+};
+
+const PolicyDetailsV2 = ({ route }) => {
+  const { policy_id: policyId } = useParams();
+  const query = usePolicyQuery2({ policyId });
+  let newData = {};
+  if (query?.data?.data) {
+    newData = {
+      profile: serialize(query.data.data),
+    };
+    newData.profile.policy.profiles = [];
+  }
```

Comment:
To make it more declarative
```suggestion
  const data = query?.data?.data ? { profile: { ...serialize(query.data.data), policy: { profiles: [] }} : {};
```